bark/world/evaluation/ltl/label_functions/ego_below_speed_limit_label_function.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2020 fortiss GmbH
+//
+// Authors: Julian Bernhard, Klemens Esterle, Patrick Hart and
+// Tobias Kessler
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#include "bark/world/evaluation/ltl/label_functions/ego_below_speed_limit_label_function.hpp"
+
+#include "bark/world/observed_world.hpp"
+
+using bark::models::dynamic::StateDefinition;
+
+namespace bark {
+namespace world {
+namespace evaluation {
+
+EgoBelowSpeedLimitLabelFunction::EgoBelowSpeedLimitLabelFunction(
```

Comment:
Why use BelowSpeedLimitLabelFunction and EgoBelowSpeedLimitLabelFunction? would this not be possible in one class?